### PR TITLE
chore(tests): Move tests/snuba/api/endpoint tests to tests/sentry

### PR DESCRIPTION
**Pull request by armenzg**
Without this change, it could sometimes be hard to find the tests since they would not match the mental mapping of `src/sentry` to `tests/sentry`.

### CODE REVIEW

File: tests/sentry/api/endpoints/test_group_details.py [hunk 1]
```diff
@@ -788,3 +793,304 @@ def test_collapse_release(self):
         assert response.data["firstRelease"] is None
         response = self.client.get(url, {"collapse": ["release"]})
         assert "firstRelease" not in response.data
+
+
+class GroupDetailsSnubaTest(APITestCase, SnubaTestCase):
```
Comment (armenzg) [inline tests/sentry/api/endpoints/test_group_details.py:798]: I've moved this from `tests/snuba/.../test_group_details.py`

File: tests/sentry/api/endpoints/test_group_event_details.py [hunk 2]
```diff
@@ -389,3 +389,118 @@ def test_query_issue_platform_title(self):
         assert response.data["id"] == str(occurrence.event_id)
         assert response.data["previousEventID"] is None
         assert response.data["nextEventID"] is None
+
+
+class GroupEventDetailsTest(APITestCase, SnubaTestCase, PerformanceIssueTestCase):
```
Comment (armenzg) [inline tests/sentry/api/endpoints/test_group_event_details.py:394]: I've moved this from `tests/snuba/.../test_group_event_details.py`

**Comment by shellmayr** [comment]
@armenzg any specific reason why this was not done after all?

**Comment by asottile-sentry** [comment]
my guess is that sns intentionally segregated these tests so make it easier to check integration in the snuba repository 🤷

**Pull request closed without merging**